docs/README.md

```diff
@@ -94,8 +94,49 @@ Download and extract the appropriate binaries from
 
 > The path where you put the binaries is referenced as `$INSTALL_DIR`
 
-You can also use the [download.sh](../download.sh) script which uses following
-environment variables as parameters:
+### Instrument a .NET application
+
+When running your application, make sure to:
+
+1. Set the [resources](config.md#resources).
+2. Set the environment variables from the table below.
+
+| Environment variable                 | .NET version           | Value                                                                               |
+|--------------------------------------|------------------------|-------------------------------------------------------------------------------------|
+| `COR_ENABLE_PROFILING`               | .NET Framework         | `1`                                                                                 |
+| `COR_PROFILER`                       | .NET Framework         | `{918728DD-259F-4A6A-AC2B-B85E1B658318}`                                            |
+| `COR_PROFILER_PATH_32`               | .NET Framework         | `$INSTALL_DIR/win-x86/OpenTelemetry.AutoInstrumentation.Native.dll`                 |
```

Comment:
What's about COR_PROFILER_PATH? It is also supported.

Reply:
Same as below.